libraries/stdlib/samples/test/samples/collections/collections.kt

```diff
@@ -271,6 +271,14 @@ class Collections {
             assertTrue(set == other, "Empty sets are equal")
             assertPrints(set, "[]")
         }
+
+        @Sample
+        fun readOnlySet() {
+            val emptySet: Set<Int> = setOf()
```

Comment:
`setOf` is already covered by the sample above. Take a look at `readOnlyList` and `singletonReadOnlyList` samples for an example.